### DISCUSSION TITLE
[ FF7 ] Crash when entering mideel

### DISCUSSION THREAD

**Discussion by jonsn0w** [julianxhokaxhiu/FFNx#326] [Q&A] [answered]
Whenever I try to enter destroyed Mideel, the game crashes. Log attached.
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/7730504/FFNx.log)

**Comment by julianxhokaxhiu** [comment, marked as answer]
You're likely using 7th Heaven and a model replacement mod. Please ask for support on their relative channel. That crash has nothing to be fixed on FFNx side. Cheers.
